### DISCUSSION TITLE
New quest for campsite type

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#6012] [Ideas]
Would you consider adding a quest for the type of campsite, i.e.[ camp_site=*](https://wiki.openstreetmap.org/wiki/Key:camp_site)? It has four possible answers; basic, standard, serviced, and deluxe.

The tag is still not very used yet but the tag camp_site will soon be added to the iD preset for tourism=camp_site: https://github.com/openstreetmap/id-tagging-schema/pull/1286

Thanks.

**Comment by matkoniecz** [comment]
first step would be researching is it matching https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md

**Reply by ERYpTION** [reply]
At first glance, I think the quest would meet most of the requirements listed. The main issue is probably the requirement "Publicly accessible on foot" since answering the quest for some campsites probably require entering the site to survey the type of facilities offered. For example to determine if the site is only of the type 'serviced' or actually 'deluxe' because it also has a "laundry and either a swimming pool or other large scale recreation facility"  I guess that for some campsites access is only for customers (does that count as publicly accessible?).

**Comment by mnalis** [comment]
> Would you consider adding a quest for the type of campsite, i.e.[ camp_site=*](https://wiki.openstreetmap.org/wiki/Key:camp_site)? It has four possible answers; basic, standard, serviced, and deluxe.

It has been considered previously, see https://github.com/streetcomplete/StreetComplete/issues/4198#issuecomment-1183073591, but decided against it in favor of more detailed tagging.

Thus, since SC user is already on the ground, StreetComplete ***already*** offers **_more details_** to be filled in, by asking for several camp-specific quests (i.e. does it have drinking water, does it have showers, does it have electricity; see https://github.com/streetcomplete/StreetComplete/pull/4213) and those typically more precisely define the camp than just  `camp_site=*`.  

So `basic`, `standard`, `serviced` are already covered by existing quests (just using more precise tags). What current quests do not cover is `deluxe`, but I (personally) find that value too useless - if I want a camp with a pool, I'd be pretty disappointed if I found out that it has tennis or basketball courts instead of a pool (which qualify for the same `deluxe` value). 

So instead of using ambiguous `camp_site=deluxe`, I'd recommend mapping those recreational amenities (i.e. swimming pool, soccer pitch, fitness centre etc) at their exact locations instead, as that is much more useful for actual users so they could know exactly what sport/leisure amenities are available (and - especially important for bigger camp sites - _where_ each of them is situated)

**Reply by ERYpTION** [reply]
Sorry that I didn't find that comment before posting the idea. Although I understand the points made, I still think type can be useful for a rough sorting of campsites. I will not leave suggestions again.

**Reply by mnalis** [reply]
> Although I understand the points made, I still think type can be useful for a rough sorting of campsites.

Perhaps :man_shrugging: , but I do not see much sense in same mapping tool asking for those more detailed quests, and then afterwards asking for less detailed quest for which answers were already given.

Perhaps if such more generic _"one tag to rule them all"_ is wanted, a recurring bot could be made to [automatically](https://wiki.openstreetmap.org/wiki/Automated_edits) add that tag to all campsites which have more detailed tags, but lack the `camp_site=*`? But I personally think it should be a job of data consumer app to consume all tags it is interested in...

>  I will not leave suggestions again.

But please, do feel free to leave them! It is only my personal opinion, and other developers might have different ones. 
Just be aware that suggestions will not always be accepted, so don't get discouraged. 
What usually helps to promote some idea is giving your specific use case, and explaining why existing way of tagging / quests are not working (or not working well) for your use case.
